go/adbc/drivermgr/adbc_driver_manager.cc

```diff
@@ -114,7 +128,175 @@ struct OwnedError {
   }
 };
 
+std::filesystem::path UserConfigDir() {
```

Comment:
This is copy-pasted from the C ADBC codebase??

Reply:
I don't think so? I haven't seen this anywhere else, is there a spot that already does this?

Reply:
ah, yea. I didn't realize this comment was on the Go copy.

Yea, we sync the code for the driver manager to the Go package so that it builds and statically links the driver manager into the Go driver_manager package. For `go get` you can only reference files that are in the actual Go module code base, so we can't just point it at the C code directly if we want it to be easily usable in other go projects.